### PR TITLE
Routing 🛣

### PR DESCRIPTION
**Pull request by revett**
### Problem

There is no routing within the current app.

### Solution

Add the following using **react-router**:

1. Page is on `/:showcaseHash`
2. Redirect `/showcase/:showcaseHash` to `/:showcaseHash`
3. Basic NotFound page

### Notes

- **Remember** to tag your merge commit after merging 👍

### CODE REVIEW

**Comment by revett** [comment]
[[SSA] React Router](https://trello.com/c/LWC40iFt/1168-ssa-react-router)

File: src/routes.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import AppContainer from './components/container/app';
+import NotFoundContainer from './components/container/not_found';
+
+export default function CreateRoutes() {
+  return (
+    <Router history={browserHistory}>
+      <Route path="/:showcaseHash" component={AppContainer} />
+      <Redirect from="/showcase/:showcaseHash" to="/:showcaseHash" />
+      <Route path="*" component={NotFoundContainer} />
```
Comment (stevenjack) [inline src/routes.js]: 500/Error page?

Comment (revett) [inline src/routes.js]: Will be done in a separate PR 👍
